beta-src/src/interfaces/map/TerritoryMapData.ts

```diff
@@ -1,16 +1,8 @@
-import {
-  AbsoluteCoordinates,
-  BBox,
-  Coordinates,
-  Label,
-  Territory,
-  Texture,
-  UnitSlot,
-} from "..";
+import { BBox, Coordinates, Label, Territory, Texture, UnitSlot } from "..";
```

Comment:
is this import correct?

Reply:
yea, just removed AbsoluteCoordinates as that interface was deleted from the codebase.